nixos/modules/installer/netboot/netboot.nix

```diff
@@ -8,6 +8,19 @@ with lib;
 {
   options = {
 
+    netboot.squashfsCompression = mkOption {
+      default = with pkgs.stdenv.hostPlatform; "xz -Xdict-size 100% "
```

Comment:
Missing type?

Reply:
Typed. (And the corresponding isoImage option.)